framework/tlvf/src/src/CmduMessageRx.cpp

```diff
@@ -51,6 +51,26 @@
 
 using namespace ieee1905_1;
 
+int CmduMessageRx::getNextTlvType() const
+{
+    if (!getCmduHeader())
+        return -1;
+    sTlvHeader *tlv = reinterpret_cast<sTlvHeader *>(msg.prevClass()->getBuffPtr());
+    return tlv->type;
+}
+
+uint16_t CmduMessageRx::getNextTlvLength() const
+{
+    if (!getCmduHeader())
+        return -1;
```

Comment:
This would need an explicit cast to uint16_t IMO.

Or maybe even better: make the return value int.

Reply:
I've changed it to return UINT16_MAX